src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -201,6 +201,98 @@ After a short (less than 5 minute) delay, the status will change from **Pending
+
+- [Cluster Administrator]({% link cockroachcloud/authorization.md %}#cluster-administrator) to add or remove private endpoint trusted owners.
+- [Cluster Operator]({% link cockroachcloud/authorization.md %}#cluster-operator) to list or get details about private endpoint trusted owners.
+
+### Add a private endpoint trusted owner
+
+To [add a private endpoint trusted owner](https://www.cockroachlabs.com/docs/api/cloud/v1#post-/api/v1/clusters/-cluster_id-/networking/private-endpoint-trusted-owners):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/networking/private-endpoint-trusted-owners \
+  --header 'Authorization: Bearer {bearer_token' \
+  --header 'content-type: application/json' \
+  --data '{"external_owner_id":"{aws_account_id}","type":"AWS_ACCOUNT_ID"}'
+~~~
+
+Replace:
+- `{cluster_id}`: The ID of the cluster.
+- `{bearer_token}`: The service account's API key.
+- `{aws_account_id}`: The ID of the AWS account to trust.
+
+Next, you can [create an AWS endpoint](#step-2-create-an-aws-endpoint).
+
+### List private endpoint trusted owners
+
+To [list private endpoint trusted owners](https://www.cockroachlabs.com/docs/api/cloud/v1#get-/api/v1/clusters/-cluster_id-/networking/private-endpoint-trusted-owners) for a cluster:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request GET \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/networking/private-endpoint-trusted-owners \
+  --header 'Authorization: Bearer {bearer_token}'
+~~~
+
+Replace:
+
+- `{cluster_id}`: The ID of the cluster.
+- `{bearer_token}`: The service account's API key.
+
+### Get details about a trusted owner
+
+To [get details about a private endpoint trusted owner](https://www.cockroachlabs.com/docs/api/cloud/v1#get-/api/v1/clusters/-cluster_id-/networking/private-endpoint-trusted-owners/-owner_id-):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request GET \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/networking/private-endpoint-trusted-owners/{owner_id} \
+  --header 'Authorization: Bearer {bearer_token}'
+~~~
+
+Replace:
+
+- `{cluster_id}`: The ID of the cluster.
+- `{owner_id}`: The ID of a trusted AWS account.
+- `{bearer_token}`: The service account's API key.
+
+### Remove a trusted owner
+
+To [remove a private endpoint trusted owner](https://www.cockroachlabs.com/docs/api/cloud/v1#delete-/api/v1/clusters/-cluster_id-/networking/private-endpoint-trusted-owners/-owner_id-):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request DELETE \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/networking/private-endpoint-trusted-owners/{owner_id} \
+  --header 'Authorization: {bearer_token}'
+~~~
+
+Replace:
+
+- `{cluster_id}`: The ID of the cluster.
+- `{owner_id}`: The ID of a trusted AWS account.
```

Comment:
I think we should make this cloud agnostic, i.e. this is the UUID of a trusted owner entry. Same goes to owner_id everywhere else.

Reply:
+1